tests/CodeGen-Binding/Xamarin.Android.LibraryProjectZip-LibBinding/java/JavaLib/library/build.gradle

```diff
@@ -2,7 +2,7 @@ apply plugin: 'com.android.library'
 
 android {
     compileSdkVersion 25
-    buildToolsVersion "25.0.2"
+    buildToolsVersion "28.0.0"
```

Comment:
Should this become a `build.gradle.in` file so that we generate it at build-time and replace `buildToolsVersion` with `$(XAPlatformToolsVersion)`?

That way the next time we bump `$(XAPlatformToolsVersion)` we won't need to remember to update `build.gradle` as well...